src/Status/Api.elm

```diff
@@ -0,0 +1,26 @@
+module Status.Api exposing (getStatus)
+
+import Utils.Api exposing (baseUrl)
+import Http
+import Status.Types exposing (StatusMsg(NewStatus), StatusResponse)
+import Json.Decode exposing (Decoder, map2, string, field, at)
+import Types exposing (Msg(MsgForStatus))
+
+
+getStatus : Cmd Msg
+getStatus =
+    let
+        url =
+            String.join "/" [ baseUrl, "status" ]
+
+        request =
+            Http.get url decodeStatusResponse
```

Comment:
Don't worry about it !

We can potentially make the poll timer configurable using `programWithFlags`. It seems to be a way to pass flags into the elm app at start-up. Potentially we can let users set a command-line flag on alertmanager, and then have the html execute js to query for those values (or write them into a templated version of `index.html`) at startup.